<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1181</width>
    <height>785</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PyTrader v0.4</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>C:/Users/parkbumjin/.designer/backup/icon.png</normaloff>C:/Users/parkbumjin/.designer/backup/icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1181</width>
      <height>761</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Tab 1</string>
     </attribute>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>950</x>
        <y>360</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>조회</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="tableWidget_2">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>140</y>
        <width>801</width>
        <height>201</height>
       </rect>
      </property>
      <property name="rowCount">
       <number>100</number>
      </property>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <column>
       <property name="text">
        <string>알고리즘</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>매수시기</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>샤프지수</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>베타지수</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>평가손익</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>수익률(%)</string>
       </property>
      </column>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>360</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>주문</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox">
      <property name="geometry">
       <rect>
        <x>830</x>
        <y>370</y>
        <width>96</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>실시간조회</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>191</width>
        <height>351</height>
       </rect>
      </property>
      <property name="title">
       <string>수동주문</string>
      </property>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>64</width>
         <height>15</height>
        </rect>
       </property>
       <property name="text">
        <string>계좌</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>90</y>
         <width>64</width>
         <height>15</height>
        </rect>
       </property>
       <property name="text">
        <string>주문</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>130</y>
         <width>64</width>
         <height>15</height>
        </rect>
       </property>
       <property name="text">
        <string>종목</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>230</y>
         <width>64</width>
         <height>15</height>
        </rect>
       </property>
       <property name="text">
        <string>종류</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>270</y>
         <width>64</width>
         <height>15</height>
        </rect>
       </property>
       <property name="text">
        <string>수량</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>310</y>
         <width>64</width>
         <height>15</height>
        </rect>
       </property>
       <property name="text">
        <string>가격</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>40</y>
         <width>111</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox_2">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>90</y>
         <width>111</width>
         <height>22</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>신규매수</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>신규매도</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>매수취소</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>매도취소</string>
        </property>
       </item>
      </widget>
      <widget class="QComboBox" name="comboBox_6">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>230</y>
         <width>111</width>
         <height>22</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>지정가</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>시장가</string>
        </property>
       </item>
      </widget>
      <widget class="QLineEdit" name="lineEdit">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>130</y>
         <width>111</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QSpinBox" name="spinBox">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>270</y>
         <width>111</width>
         <height>22</height>
        </rect>
       </property>
       <property name="maximum">
        <number>100000</number>
       </property>
      </widget>
      <widget class="QSpinBox" name="spinBox_2">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>310</y>
         <width>111</width>
         <height>22</height>
        </rect>
       </property>
       <property name="maximum">
        <number>1000000000</number>
       </property>
       <property name="singleStep">
        <number>50</number>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_2">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>170</y>
         <width>111</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(170, 255, 255)</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>10</y>
        <width>831</width>
        <height>351</height>
       </rect>
      </property>
      <property name="title">
       <string>잔고 및 보유현황</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>410</y>
        <width>831</width>
        <height>301</height>
       </rect>
      </property>
      <property name="title">
       <string>자동매매 리스트</string>
      </property>
      <widget class="QTableWidget" name="tableWidget_3">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>30</y>
         <width>791</width>
         <height>261</height>
        </rect>
       </property>
       <property name="rowCount">
        <number>1</number>
       </property>
       <row/>
       <column>
        <property name="text">
         <string>주문유형</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>새 행</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>종목명</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>호가구분</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>수량</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>상태</string>
        </property>
       </column>
      </widget>
     </widget>
     <widget class="QTableWidget" name="tableWidget">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>30</y>
        <width>801</width>
        <height>101</height>
       </rect>
      </property>
      <property name="rowCount">
       <number>1</number>
      </property>
      <row/>
      <column>
       <property name="text">
        <string>예수금(D+2)</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>총매입금액</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>총평가금액</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>총평가 손익</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>추정자산</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>총수익률(%)</string>
       </property>
      </column>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>10</y>
        <width>331</width>
        <height>711</height>
       </rect>
      </property>
      <property name="title">
       <string>알고리즘1</string>
      </property>
      <widget class="QTableWidget" name="tableWidget_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>311</width>
         <height>461</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>311</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>311</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="rowCount">
        <number>1</number>
       </property>
       <row/>
       <column>
        <property name="text">
         <string>종목명</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>수익률</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>총금액</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>수량</string>
        </property>
        <property name="font">
         <font>
          <pointsize>7</pointsize>
         </font>
        </property>
       </column>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_5">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>10</y>
        <width>331</width>
        <height>711</height>
       </rect>
      </property>
      <property name="title">
       <string>알고리즘1</string>
      </property>
      <widget class="QTableWidget" name="tableWidget_5">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>311</width>
         <height>461</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>311</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>311</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="rowCount">
        <number>1</number>
       </property>
       <row/>
       <column>
        <property name="text">
         <string>종목명</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>수익률</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>총금액</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>수량</string>
        </property>
        <property name="font">
         <font>
          <pointsize>7</pointsize>
         </font>
        </property>
       </column>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_6">
      <property name="geometry">
       <rect>
        <x>830</x>
        <y>10</y>
        <width>331</width>
        <height>711</height>
       </rect>
      </property>
      <property name="title">
       <string>알고리즘1</string>
      </property>
      <widget class="QTableWidget" name="tableWidget_6">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>311</width>
         <height>461</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>311</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>311</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="rowCount">
        <number>1</number>
       </property>
       <row/>
       <column>
        <property name="text">
         <string>종목명</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>수익률</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>총금액</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>수량</string>
        </property>
        <property name="font">
         <font>
          <pointsize>7</pointsize>
         </font>
        </property>
       </column>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>쪽</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>쪽</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1181</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>파일</string>
    </property>
   </widget>
   <addaction name="menu"/>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
